### ISSUE TITLE
Hybrid w Hydrogen Work

### ISSUE THREAD

**Issue by grantbuster** [NREL/reV#338]
**Feature description:**
We want to model hybrid wind+solar timeseries power generation with curtailed electricity going to an electrolyzer. We then want to know LCOE and LCOH for the hybrid plant, with LCOT-H (cost of hydrogen transmission) for the hybrid plant. Here is the proposed workflow with (new) development tasks:

1. Run separate reV pipelines for wind and solar. Use the same supply curve aggregation resolution to get them onto the same grid.
2. Run a (new) hybridization reV module that would compile the hybrid power generation profiles into a single power profile and would make a composite supply curve table. Include optional inputs for max wind/solar capacity or bounds on wind/solar capacity ratio.
3. Run NRWAL with a (new) electrolyzer model that converts curtailed power to hydrogen. Create a curtailed power gen profile and a hydrogen production timeseries profile. Input spatially varying distance-to-load-center values to NRWAL to compute a LCOT-H value that is a function of transmission distance, hydrogen production, and h2 capacity size. 
4. Run parametrics (reV-batch) on hybrid plant sizes and ratios between wind/solar capacities.
5. Make maps, make money. 

To summarize, the reV development tasks are:

1. New hybridization module (takes in two supply curve tables, makes hybrid power gen profiles and new composite supply curve tables)
2. Rename the reV-offshore module to reV-nrwal and make sure it can run on timeseries profile inputs.
3. Build an electrolyzer and LCOT-H model in NRWAL (do this in NRWAL yaml configs, not in reV code base).
4. Make sure the steps above can be run as a reV pipeline from CLI's and configs so we can use reV-batch

**Charge code**
10348 05.01.04

**Urgency / Timeframe**
Needed for Q2 milestone. reV dev needs to be done early so we can actually run tests and make maps.

**Comment by grantbuster** [comment]
Priority is to develop and test the NRWAL module on timeseries data so that we can run a prototype wind+h2 pipeline (no solar)

**Comment by grantbuster** [comment]
More general NRWAL integration complete in #342

**Comment by ppinchuk** [comment]
New hybridization module addedd in #350
